docs-developer/src/doc/docs/developer_guide/plugin-development/introduction.md

```diff
@@ -22,7 +22,7 @@ It has pre-configured dependencies, publishing and signing of your artifacts.
 
 ### Manual
 
-At a bare minimum, Dokka requires `Kotlin Gradle Plugin` and `dokka-core` dependencies:
+At a bare minimum, a Dokka plugin requires Kotlin Gradle Plugin and `dokka-core` as dependencies:
```

Comment:
`Kotlin Gradle Plugin as dependency` can be non-obvious of what it means.

Reply:
Removed this part